README.md

```diff
@@ -162,7 +162,102 @@ MODELS=`[
 
 You can change things like the parameters, or customize the preprompt to better suit your needs. You can also add more models by adding more objects to the array, with different preprompts for example.
 
-#### Custom prompt templates:
+##### OpenAI API compatible models
+
+Chat UI can be used with any API server that supports OpenAI API compatibility, for example [oobabooga](https://github.com/oobabooga/text-generation-webui/tree/main/extensions/openai), [LocalAI](https://github.com/go-skynet/LocalAI), [FastChat](https://github.com/lm-sys/FastChat/blob/main/docs/openai_api.md), [llama-cpp-python](https://github.com/abetlen/llama-cpp-python), and [ialacol](https://github.com/chenhunghan/ialacol).
+
+This makes Chat UI works with [oobabooga](https://github.com/oobabooga/text-generation-webui/tree/main/extensions/openai).
+
+```
+MODELS=`[
+  {
+    "name": "oobabooga",
+    "id": "oobabooga",
+    "userMessageToken": "### User:\n",
+    "userMessageEndToken": "\n", # Optional
+    "assistantMessageToken": "### Assistant:\n",
+    "preprompt": "You are ai assistant pretend to be a hugging face model", # Will be used as the system prompt
+    "promptExamples": [],
+    "parameters": { # These will be sent to the OpenAI API compatible server
+      "temperature": 0.9,
+      "top_p": 0.95,
+      "repetition_penalty": 1.2,
+      "top_k": 50,
+      "truncate": 1000,
+      "max_new_tokens": 1024,
+      "stop": ["###"]
```

Comment:
This lead to the message to stop when hitting a subsubtitle in markdown, this is quite common